hoodie-common/src/main/java/com/uber/hoodie/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -119,21 +118,21 @@ private void readObject(java.io.ObjectInputStream in)
   }
 
   /**
-   * Get all instants (commits, delta commits, compactions) that produce new data, in the active
+   * Get all instants (commits, delta commits) that produce new data, in the active
    * timeline *
    */
-  public HoodieTimeline getCommitsAndCompactionsTimeline() {
+  public HoodieTimeline getCommitsTimeline() {
```

Comment:
Compactions didn't make sense here so renaming this to just `getCommitsTimeline()`, there is another method called `getCommitTimeline()` which returns timeline filtered only by `.commit` action. Ideally, we could just combine both of these to one since COW will anyways never see .deltacommit but I'm skeptical of changing the old method in case it may cause regressions.

Reply:
can we investigate that more.. this is a good chance to cleanup.do you see a reason at all any part of code needs to differentiate between a normal commit and a compaction commit. ?

Reply:
Okay I think this was the start of the conversation around this. There is a fair bit of cleanup needed here.. so lets just keep it as you suggested `getCommitsTimeline()` and move forward..